services/slam/server_test.go

```diff
@@ -29,9 +34,24 @@ import (
 const (
 	testSlamServiceName  = "slam1"
 	testSlamServiceName2 = "slam2"
+	chunkSizeServer      = 100
 )
 
-func TestServer(t *testing.T) {
+type StreamServerMock struct {
+	grpc.ServerStream
+	rawBytes []byte
```

Comment:
Could also avoid doing concatenation here by storing results in a [][]byte.  however, I don't feel this increases the efficacy of the test.